doc/aliases/add_toggle.html

```diff
@@ -0,0 +1,14 @@
+<script type="text/javascript">
+/*
+  Creates a toggleable div that includes
+  the text/code associated to a button.
+*/
+var $getDiv = $('div.newInnerHTML').last();
+var buttonName = $getDiv.html();
+var label = getLabelName(buttonName.trim());
```

Comment:
I suggest adding predefined command aliases for all languages: `addtoggle_cpp, addtoggle_java, addtoggle_py` to avoid such situations.